roles/docker/tasks/main.yml

```diff
@@ -62,6 +62,23 @@
   with_items: "{{ docker_package_info.pkgs }}"
   when: (ansible_os_family != "CoreOS") and (docker_package_info.pkgs|length > 0)
 
+- name: Set docker daemon options
+  template:
+    src: docker-opts.j2
+    dest: "/etc/default/docker"
+    owner: root
+    group: root
+    mode: 0644
+  when: ansible_os_family != "CoreOS"
+
+- name: Create legacy EnvironmentFile path
+  file: path=/etc/sysconfig state=directory mode=0755
```

Comment:
As @mattymo said, that is the default. I could add a when condition to only create the dir when it doesn't exist.